cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -826,3 +827,26 @@ class EditTags(CFMENavigateStep):
 
     def step(self, *args, **kwargs):
         self.prerequisite_view.policy.item_select('Edit Tags')
+
+
+class SetOwnershipView(ServicesCatalogView):
+    submit = Button("Submit")
+    reset = Button("Reset")
+    cancel = Button("Cancel")
+    @View.nested
+    class form(View):
+        select_an_owner = ReactSelect("user_name")
+        select_group = ReactSelect("group_name")
+
+    @property
+    def is_displayed(self):
+        return self.form.select_an_owner.is_displayed and self.form.select_group.is_displayed
```

Comment:
**Optional** - You might want to add other checks such as `self.submit.is_displayed and self.cancel.is_enabled and self.reset.is_enabled`.

Reply:
Done

Reply:
Done, Can't add `is_display` for *`reset`* button before changing the state.